### ISSUE TITLE
Unable to launch the app

### ISSUE THREAD

**Issue by Y-PLONI** [yoshiask/FluentStore#84]
### Before you start...

- [X] Have you checked if similar issue have already been reported?

### What windows are you seeing the problem on?

Windows 11

### What OS/App version/architecture are you seeing the problem on?

24H2
The latest app version published on GitHub

### Description

I turned it on, and... nothing!
In the task manager you can see that the process "Sending Windows errors" is running for a moment.
That's how, once I run the app, it doesn't start at all!

### Expected behavior

to work as usual

**Comment by yoshiask** [comment]
Howdy! Sorry you're experiencing this issue. Are there any error messages in the Applications section of Event Viewer?

**Comment by Y-PLONI** [comment]
Where should I see an error?
The app does not load at all.
There is a process in the task manager, please guide me where I can see the errors.

**Comment by yoshiask** [comment]
[0.4.0 Beta](https://github.com/yoshiask/FluentStore/releases/tag/v0.4.0-beta) has been released, are you still experiencing issues with that version?

**Comment by Y-PLONI** [comment]
@yoshiask 
YES

**Comment by Y-PLONI** [comment]
`[-](https://github.com/yoshiask/FluentStore/issues/84#) <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
[-](https://github.com/yoshiask/FluentStore/issues/84#) <System>
  <Provider Name="Application Error" Guid="{a0e9b465-b939-57d7-b27d-95d8e925ff57}" /> 
  <EventID>1000</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x8000000000000000</Keywords> 
  <TimeCreated SystemTime="2025-03-20T08:34:50.6524426Z" /> 
  <EventRecordID>196904</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="2956" ThreadID="14396" /> 
  <Channel>Application</Channel> 
  <Computer>מחשב-בית</Computer> 
  <Security UserID="S[-](https://github.com/yoshiask/FluentStore/issues/84#)1-5-21-3189552838-987810334-1112634709-1001" /> 
  </System>
- <EventData>
  <Data Name="AppName">FluentStore.App.exe</Data> 
  <Data Name="AppVersion">0.4.0.0</Data> 
  <Data Name="AppTimeStamp">67890000</Data> 
  <Data Name="ModuleName">Microsoft.UI.Xaml.dll</Data> 
  <Data Name="ModuleVersion">3.1.6.0</Data> 
  <Data Name="ModuleTimeStamp">41a6fb67</Data> 
  <Data Name="ExceptionCode">c000027b</Data> 
  <Data Name="FaultingOffset">0000000000009125</Data> 
  <Data Name="ProcessId">0x2c08</Data> 
  <Data Name="ProcessCreationTime">0x1db9972f1e2aed9</Data> 
  <Data Name="AppPath">C:\Program Files\WindowsApps\52374YoshiAskharoun.FluentStore_0.4.0.0_x64__7sdtne7r32rbe\FluentStore.App.exe</Data> 
  <Data Name="ModulePath">C:\Program Files\WindowsApps\Microsoft.WindowsAppRuntime.1.6_6000.424.1611.0_x64__8wekyb3d8bbwe\Microsoft.UI.Xaml.dll</Data> 
  <Data Name="IntegratorReportId">82e65f31-8517-4bb2-9c35-a4b695de2d63</Data> 
  <Data Name="PackageFullName">52374YoshiAskharoun.FluentStore_0.4.0.0_x64__7sdtne7r32rbe</Data> 
  <Data Name="PackageRelativeAppId">App</Data> 
  </EventData>
  </Event>`

**Comment by yoshiask** [comment]
Can you walk me through how you installed Fluent Store?

**Comment by Y-PLONI** [comment]
FluentStoreBeta_Installer_x64.exe

‫בתאריך יום ד׳, 26 במרץ 2025 ב-16:09 מאת ‪yoshiask‬‏ <‪
***@***.***‬‏>:‬

**Comment by yoshiask** [comment]
Just to make sure, how are you launching the app?

**Comment by Y-PLONI** [comment]
Via the Start Menu
Would you like a screenshot or something?

I have an idea: Does the app try to access the internet before it starts? I
use a content filtering service that operates on a whitelist.

‫בתאריך יום ד׳, 26 במרץ 2025 ב-16:27 מאת ‪yoshiask‬‏ <‪
***@***.***‬‏>:‬

**Comment by yoshiask** [comment]
Yes, Fluent Store does access the internet. Release 0.4.0 directly makes requests to [dist.ipfs.tech](https://dist.ipfs.tech) for bootstrapping IPFS, I believe you can get around this by setting up an IPFS node yourself. That being said, I'm surprised it outright fails to start, even before the splash screen appears. I believe something else is going on here.

**Comment by Y-PLONI** [comment]
Do you have an idea? No error checking logs?
